. editorconfig

```diff
@@ -0,0 +1,28 @@
+# https://EditorConfig.org
+
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 4
+
+[Makefile]
+indent_style = tab
+
+[make.bat]
+indent_style = tab
+
```

Comment:
Since we don't have any of these (and please never will, especially `make.bat`), should we include them at all?

Reply:
Those (including `make.bat`) are generated by sphinx

Reply:
Ah forgot about that, disregard.